### DISCUSSION TITLE
Customizing theme

### DISCUSSION THREAD

**Discussion by imadegautama** [irsyadadl/intentui#210] [Q&A]
Hi!, i am curious about this libs. Is there any method to change the theme generally like shadcn did?. Thankyou

**Comment by irsyadadl** [comment]
This wasn't originally planned, but due to high demand, I'm now considering the best way to include it on the theme page. Don't have ETA. But if someone can contribute to it would be very appreciated.

**Comment by irsyadadl** [comment]
@Tamagouuu Updated theme page, now you can switch different theme. Also note that, PR would be appreciated. https://getjustd.com/themes
